requirements/extra.txt

```diff
@@ -0,0 +1,15 @@
+# Extra dependencies required for development and tests
+
+-r base.txt
+
+# Pytest and the optional pytest coverage plugin for tests
+pytest
+pytest-cov
+
+# Optional dependencies for extra functionality
+sqlalchemy
+scikit-image
```

Comment:
does requiring scikit-image imply we need to require networkx seeing how scikit-image will not install it itself.